### PR TITLE
Add more supported attributes to MarkdownText

### PR DESCRIPTION
**Pull request by THEAccess**
This PR adds the following attributes (parameters) to MarkdownText: textColor, fontSize, letterSpacing, textAlign, maxLines, style, Update README.

### CODE REVIEW

File: markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/MarkdownText.kt [hunk 1]
```diff
@@ -19,22 +32,60 @@ import io.noties.markwon.linkify.LinkifyPlugin
...
+            TextAlign.Left -> View.TEXT_ALIGNMENT_TEXT_START
+            TextAlign.Right -> View.TEXT_ALIGNMENT_TEXT_END
+            TextAlign.Center -> View.TEXT_ALIGNMENT_CENTER
+            TextAlign.Justify -> null
+            TextAlign.Start -> View.TEXT_ALIGNMENT_TEXT_START
+            TextAlign.End -> View.TEXT_ALIGNMENT_TEXT_END
+            null -> null
+        }?.let {
+            textAlignment = it
+        }
```
Comment (jeziellago) [inline markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/MarkdownText.kt:75]: Should be simplified:
```suggestion
        when (textAlign) {
            TextAlign.Left, TextAlign.Start -> View.TEXT_ALIGNMENT_TEXT_START
            TextAlign.Right, TextAlign.End -> View.TEXT_ALIGNMENT_TEXT_END
            TextAlign.Center -> View.TEXT_ALIGNMENT_CENTER
            else -> null
        }?.let { align ->
            textAlignment = align
        }
```

File: markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/MarkdownText.kt [hunk 2]
```diff
@@ -19,22 +32,60 @@ import io.noties.markwon.linkify.LinkifyPlugin
...
+    )
+
     val context = LocalContext.current
     val markdownRender = remember { createMarkdownRender(context) }
     val markdownText = TextView(LocalContext.current).apply {
-        setTextColor(textColor)
+        setTextColor(textColor.toArgb())
+        setMaxLines(maxLines)
+        setTextSize(TypedValue.COMPLEX_UNIT_DIP, mergedStyle.fontSize.value)
+        if(mergedStyle.letterSpacing.type != TextUnitType.Unspecified) setLetterSpacing(mergedStyle.letterSpacing.value)
```
Comment (jeziellago) [inline markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/MarkdownText.kt:64]: `setLetterSpacing` only accepts value in `em` unit.
If the `type` in `mergedStyle.letterSpacing.type` isn't `TextUnitType.Em`, the letterSpacing will be wrong.

File: README.md [hunk 3]
```diff
@@ -38,6 +38,19 @@ fun ExampleContent() {
...
+Most of the attributes of that a default `Text` composable has are also supported by `MarkdownText`. 
+
+- textColor 
+- fontSize 
+- letterSpacing 
+- textAlign
+- maxLines
+- style (only styles for supported attributes is applied)
+
+The font can be changed by passing a font xml resource as `fontResource` parameter.
```
Comment (jeziellago) [inline README.md:52]: Could you put a code snippet here (as an example)?

**Review by jeziellago — CHANGES_REQUESTED** [review]

File: markdowntext/build.gradle [hunk 4]
```diff
@@ -66,9 +66,9 @@ afterEvaluate {
             release(MavenPublication) {
                 from components.release
 
-                groupId = 'dev.jeziellago'
+                groupId = 'com.luminarlab.ui'
                 artifactId = 'compose-markdown'
-                version = '0.1.0'
+                version = '1.0.0-alpha01'
```
Comment (jeziellago) [inline markdowntext/build.gradle:71]: Publication configs mustn't be modified.

Comment (THEAccess) [reply markdowntext/build.gradle:71]: Must have slipped through. Was for internal purposes only. I'll revert that

**Comment by jeziellago** [comment]
Update your branch with `0.2.0` to solve conflicts in `sample/src/main/java/dev/jeziellago/compose/markdown/MainActivity.kt`

**Review by jeziellago — APPROVED** [review]

**Pull request merged**